Detector/DetFCChhTrackerTkLayout/src/TkLayoutEndcap_Geo.cpp

```diff
@@ -16,7 +19,7 @@ static DD4hep::Geometry::Ref_t createTkLayoutTrackerEndcap(DD4hep::Geometry::LCD
   // shorthands
   DD4hep::XML::DetElement xmlDet = static_cast<DD4hep::XML::DetElement>(xmlElement);
   Dimension dimensions(xmlDet.dimensions());
-  double l_overlapMargin = 0.00001;
+  double l_overlapMargin = 0.01;
```

Comment:
Why did the margin change? The actual geometry should've stayed unchanged?

Same for the barrel: Why was it now possible to remove the margin while still passing the overlap checks?